product_docs/docs/biganimal/release/getting_started/01_preparing_azure/index.mdx

```diff
@@ -0,0 +1,203 @@
+---
+title: "Preparing your Azure account"
+
+---
+
+BigAnimal requires you to configure your Azure subscription, before you deploy your clusters. The configurations that you perform ensure that your Azure subscription is prepared to meet your clusters' resource requirements, such as: 
+
+ * Is there a sufficient limit on the number vCPU or Public IP addresses in your region?
+ * Is there a restriction on VM size SKUs for the Standard Dsv2 and Standard Esv3 VM size families?
```

Comment:
We don't use `Dsv2` anymore.

Reply:
Removed mention of the Dsv2 family here.

The change in VM family would reflect in the Azure script output as well. 
@xugy99 i do not have access to the dev environment. Can you provide me with the updated outputs of the azure script so that I can refresh the doc here https://61aa29eb71b4c06371f6c489--edb-docs-staging.netlify.app/docs/biganimal/latest/getting_started/01_preparing_azure/

Reply:
the tricky part is we are refactoring this script. The arguments and output might change totally. Should we wait for a while until the new version of this script comes out?

Reply:
@drothery-edb what are your thoughts?

Reply:
@Simonzhaohui when do you expect the script to be ready? is there a ticket tracking this work?